CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ Windows users should use `gradlew.bat` instead.
 ```
 $ cd /path/to/git/clone/of/ArchUnit
 $ ./gradlew showJdkVersion
-Configured JDK: 1.9
+Configured JDK: 11
```

Comment:
For the sake of maintainability, what about just making it 14? :rofl: I tought you're working on having a separate test source set for Record tests which will then require the whole build to be at least JDK 14? :wink: